client/src/components/application/Transactions.vue

```diff
@@ -402,9 +405,6 @@
   .name-option {
     margin-top: 4px;
   }
-  .name-option.accepted {
-    color: var(--cyan);
```

Comment:
`getNameClasses()` now returns "c-cyan" instead of "accepted", which of course is now a global style.